py_mind_memo/graphics.py

```
@@ -280,27 +308,40 @@ def _draw_rich_text(self, x, y, node, base_font, tags):
             
         return item_ids
 
-    def _draw_node_image(self, x, y, text_block_w, total_h, node, tags) -> tuple:
-        """ノードに設定された画像をテキスト上部に描画し、占有した高さと幅のオフセットを返す (w_offset, h_offset)"""
+    def _draw_node_media(self, x, y, text_block_w, first_line_w, total_h, node, tags) -> tuple:
```

Ruff (0.15.7)

[warning] Unused method argument: `text_block_w`

(ARG002)